sdk/messaging/azservicebus/internal/amqpLinks.go

```diff
@@ -176,12 +173,7 @@ func (links *AMQPLinksImpl) recoverLink(ctx context.Context, theirLinkRevision L
 	}
 
 	// best effort close, the connection these were built on is gone.
-	if err := links.closeWithoutLocking(ctx, false); err != nil {
-		if GetRecoveryKind(err) == RecoveryKindConn {
-			return err
-		}
-	}
-
+	_ = links.closeWithoutLocking(ctx, false)
```

Comment:
Does `closeWithoutLocking` need to return an error?  Looks like it's ignored in other places too.

Reply:
_need_ is probably a good question but I think it's best to return relevant info and then let the caller sort it out. (just my personal preference)